server/sonar-server/src/main/java/org/sonar/server/authentication/ws/AuthenticationWs.java

```diff
@@ -61,4 +62,12 @@ private static void defineLoginAction(NewController controller) {
       .setRequired(true);
   }
 
+  private static void defineLogoutAction(NewController controller) {
```

Comment:
We put the login WS public, so logout should also follow the same policy.
An external portal could use these WS to create a token (Cookies) and reuse them to avoid having to enter login/password everytime